tests/test_core.py

```diff
@@ -1,22 +1,14 @@
 """Tests standard tap features using the built-in SDK tests library."""
 
-import datetime
-
 from singer_sdk.testing import get_tap_test_class
 
 from tap_file.tap import TapFile
 
-SAMPLE_CONFIG = {
-    "start_date": datetime.datetime.now(datetime.timezone.utc).strftime("%Y-%m-%d"),
-    # TODO: Initialize minimal tap config
-}
+SAMPLE_CONFIG = {}
```

Comment:
We should get the basic tests running for this tap, we can setup an example S3 bucket with credentials (I can put them in the Repo, eventually we can migrate this to be Meltano's own S3 creds we'll just make a seperate issue and we can use AutoIDM's for now)

Reply:
Will work on this after I complete tests for target-posgres.